### ISSUE TITLE
View jumping downwards when selecting the cursor below column split

### ISSUE THREAD

**Issue by jasht1** [ckRobinson/multi-column-markdown#87]
I wanted to start off by thanking @ckRobinson for a much-needed plugin with great usability & documentation. I've seen that this issue has been mentioned in another post but thought it deserved its own issue page. below I will detail my expirence of it on:
- Windows 10
- obsidian v1.3.5
- multi-column v0.8.3

Without multicolumn page behaves as expected, view only moving when selecting at top & bottom edge & scrolling as seen here:
https://github.com/ckRobinson/multi-column-markdown/assets/44931733/29c1b6fe-cc16-4bde-a5d1-517326d2e7c5
with a multicolumn split: page jumps downward by roughly the height of the split section. This seems to happen when:
- text cursor is selected with mouse at points below the split section
- dosent happen every time
https://github.com/ckRobinson/multi-column-markdown/assets/44931733/b0de1bd0-b724-4178-8dbb-0d676bd11a06

with the latest update, i have noticed the behaviour change slightly:
- often times instead of previous behaviour the screen flashes to a lower point and back up which I did not expirence previously
- previous behaviour still occurs, but jumps now seem proportional to the vertical distance between the previous cursor position and selected position.

**Comment by morgan-j-black** [comment]
Same for me  🙂

Obsidian v1.4.8
MCM v0.8.3
Windows 11 Home

Thanks!

**Comment by eofferma** [comment]
Same here

Obsidian v1.4.13
MCM v0.8.3
Ubuntu 22.04.3 LTS

Thank you!

**Comment by cday960** [comment]
I found if you use fenced divs the issue goes away.

My setup went from:

`````
```start-multi-col
number of cols: 2
shadow: false
border: false
```
`````

to
```
::::: {.columns colcount=2 border=false shadow=false}

<column content>

:::::
```

This fixed my issue. Hopefully it works for you as well.

Obsidian v.1.4.16
Windows 10 Pro

**Comment by ckRobinson** [comment]
This issue has been fixed in an upcoming release, that I hope to get out soon(*tm*). The core of this issue is basically a game of tug-of-war MCM is having with Obsidian and as cday960 said, it doesn't happen with pandoc syntax. In order to put the issue to bed there is going to be a syntax update. I tried my absolute hardest to avoid updating the syntax but it was the only way. I have added a command to the upcoming release that will automatically find and update everything for you to make the transition as smooth as possible though.
